ncm-network/src/main/pan/components/network/core-schema.pan

```diff
@@ -100,7 +100,7 @@ type structure_interface = {
     "driver"  ? string
     "bootproto" ? string
     "onboot" ? string
-    "type"    ? string with match(SELF, '^(Ethernet|Bridge|Tap|xDSL)$')
+    "type"    ? string with match(SELF, '^(Ethernet|Bridge|Tap|xDSL|OVSBridge|OVSPort)$')
```

Comment:
Hi,
I may add OVS_EXTRA, but I cannot test it yet (we are not using it at IPHC).
Jerome